SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialSender.h

```diff
@@ -84,6 +84,7 @@ class SPATIALGDK_API USpatialSender : public UObject
 	void SendEmptyCommandResponse(Worker_ComponentId ComponentId, Schema_FieldId CommandIndex, Worker_RequestId RequestId);
 	void SendAddComponent(USpatialActorChannel* Channel, UObject* Subobject, const FClassInfo& Info);
 	void SendRemoveComponent(Worker_EntityId EntityId, const FClassInfo& Info);
+	void SendInterestComponentChange(Worker_EntityId EntityId, uint32_t OldComponent, uint32_t NewComponent);
```

Comment:
Worker_ComponentId instead of uint32_t ?

Reply:
Also I think we want these as const args. I think it is the style we are adopting in the gdk.